docs/envs/shadow_dexterous_hand/index.md

```diff
@@ -7,7 +7,7 @@ lastpage:
 
 These environments are based on the [Shadow Dexterous Hand](https://www.shadowrobot.com/), 5 which is an anthropomorphic robotic hand with 24 degrees of freedom. Of those 24 joints, 20 can be controlled independently whereas the remaining ones are coupled joints.
 
-* `HandReach-v1`: ShadowHand has to reach with its thumb and a selected finger until they meet at a desired goal position above the palm.
+* `HandReach-v2`: ShadowHand has to reach with its thumb and a selected finger until they meet at a desired goal position above the palm.
 * `HandManipulateBlock-v1`: ShadowHand has to manipulate a block until it achieves a desired goal position and rotation.
```

Comment:
`HandManipulateBlock` `HandManipulateEgg` `HandManipulatePen` need to be verified that they are not changed

Reply:
I ran the reset tests and all of those environments passed without the new patch. Therefore, their behavior should be unchanged. This is what I was referring to in my previous message when I said several environments are not affected and should therefore not be bumped.

Reply:
Still, since the code was changed it will have to pass additional validation to ensure it was unchanged with `check_environments_match`,
 something like this: https://github.com/Kallinteris-Andreas/gymnasium-changes-validation/tree/main/camera_id_refactor (you can do it for 100k steps, feel free to reset one of the environments prior to using `check_environments_match`)
 
 Note: do not add a test in this project, do it in a separate repo
 
 Thanks! (note: this is most likely the last code related request I will make)

Reply:
```
from gymnasium.utils.env_match import check_environments_match

from robotics_old.gymnasium_robotics.envs.shadow_dexterous_hand.manipulate_block import MujocoHandBlockEnv
from robotics_old.gymnasium_robotics.envs.shadow_dexterous_hand.manipulate_egg import MujocoHandEggEnv
from robotics_old.gymnasium_robotics.envs.shadow_dexterous_hand.manipulate_pen import MujocoHandPenEnv

from robotics_new.gymnasium_robotics.envs.shadow_dexterous_hand.manipulate_block import MujocoHandBlockEnv as NewMujocoHandBlockEnv
from robotics_new.gymnasium_robotics.envs.shadow_dexterous_hand.manipulate_egg import MujocoHandEggEnv as NewMujocoHandEggEnv
from robotics_new.gymnasium_robotics.envs.shadow_dexterous_hand.manipulate_pen import MujocoHandPenEnv as NewMujocoHandPenEnv


def check_envs_match(env_old, env_new, num_steps):
    print(f"Checking {env_old} and {env_new} match for {num_steps} steps")
    env_old.reset(seed=42)
    for _ in range(42):
        action = env_old.action_space.sample()
        env_old.step(action)
    env_old.reset(seed=42)
    env_new.reset(seed=42)
    check_environments_match(env_old, env_new, num_steps)
    print(f"{env_old} and {env_new} match (tested for {num_steps} steps)")


if __name__ == '__main__':
    NUM_STEPS = 100_000
    check_envs_match(MujocoHandBlockEnv(), NewMujocoHandBlockEnv(), NUM_STEPS)
    check_envs_match(MujocoHandEggEnv(), NewMujocoHandEggEnv(), NUM_STEPS)
    check_envs_match(MujocoHandPenEnv(), NewMujocoHandPenEnv(), NUM_STEPS)

```

passed all checks.